src/pages/OtpEmailPage/OtpEmailPage.tsx

```
@@ -44,27 +46,48 @@ export default function OtpEmailPage() {
+    console.log("Submitting OTP:", otp.join(""));
+    navigate(ROUTES.CHANGE_PASSWORD, { state: { email } });
+    return;
     const otpCode = otp.join("");
 
-    if (otpCode.length !== 6) {
+    if (otpCode.length !== 4) {
       return;
     }
 
     setIsSubmitting(true);
+    setSubmitError(null);
```

Biome (2.3.13)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)